src/t2iapi/context/service.proto

```diff
@@ -151,4 +151,12 @@ service ContextService {
    */
   rpc AssociatePatientWithAutoGeneratedPatientId (google.protobuf.Empty)
       returns (t2iapi.context.CreateContextStateWithAssociationResponse);
+  
+  /*
+  Create or adapt a context state for the given descriptor handle with the starting @ContextAssociation value
+  given as "start_association" (to trigger a context state report) and perform a transition of the previously
```

Comment:
If the context is already in that association and is only being adapted, how does it trigger a report? This doesn't make sense.

Reply:
done